File: source/aoBlogs2/Models/BlogCategoryGroupRulesModel.vb

Imports Contensive.Models.Db

Namespace Models
    Public Class BlogCategoryGroupRulesModel
        Inherits DbBaseModel
        '
        '====================================================================================================
        ''' <summary>
        '''table definition
        '''</summary>
        Public Shared ReadOnly Property tableMetadata As DbBaseTableMetadataModel = New DbBaseTableMetadataModel("Blog Category Group Rules", "ccBlogCategoryGroupRules", "default", False)
        '
        '====================================================================================================
        ' -- instance properties
        '
        Public Property BlogCategoryID As Integer
        Public Property GroupID As Integer
        '
    End Class
End Namespace

File: source/aoBlogs2/Models/Db/BlogCategoryGroupRulesModel.cs

using Contensive.Models.Db;

namespace Contensive.Blog.Models {
    public class BlogCategoryGroupRulesModel : Contensive.Models.Db.DbBaseModel {
        // 
        // ====================================================================================================
        public static DbBaseTableMetadataModel tableMetadata { get; private set; } = new DbBaseTableMetadataModel("Blog Category Group Rules", "ccBlogCategoryGroupRules", "default", false);
        // 
        // ====================================================================================================
        // -- instance properties
        // instancePropertiesGoHere
        public int BlogCategoryID { get; set; }
        public int GroupID { get; set; }
        // 
    }
}